client/src/components/Navbar.js

```diff
@@ -238,6 +238,13 @@ const Navbar = () => {
           >
             Search
           </Link>
+
+          <Link
+            to="/discover"
+            className="px-12 text-pipelines-gray-500 font-light uppercase"
+          >
+            Discover
+          </Link>
```

Comment:
You need to us "md:hidden" to make sure this doesn't show up in the mobile nav bar. Also add the link to the hamburger menu in the mobile bar